src/core/xtc.ml

```diff
@@ -28,7 +28,7 @@ let status : sym -> symb_status = fun s ->
     | _         -> Object_level
   in is_arrow_kind !(s.sym_type) false
 
-(** [print_term oc p] outputs XTC format corresponding to the term [t], to
+(** [print_term oc p] outputs XTC format corresponding to the term [t], to - @PROBLEM?
```

Comment:
The signature doesn't match.

Reply:
Hello @GuillaumeGen . Could you please explain what is this int argument everywhere? It looks really strange. Would it be possible to remove side effects? And there seems to be a bug in https://github.com/Deducteam/lambdapi/blob/82dccc5074bd42574d8d21b9b39527052d2014cd/src/core/xtc.ml#L49-L50 as what is output is i and not j...